PWGHF/D2H/Tasks/taskDs.cxx

```diff
@@ -433,7 +415,7 @@ struct HfTaskDs {
     }
   }
 
-  template <int decayChannel, bool useMl, bool fillCent, typename CandsDs>
+  template <int decayChannel, int centDetector, bool useMl, typename CandsDs>
```

Comment:
```suggestion
  template <FinalState decayChannel, o2::aod::hf_collision_centrality::CentralityEstimator centEstimator, bool useMl, typename CandsDs>
```
Rename `finalState` accordingly.
You can consider adding `using CentralityEstimator = o2::aod::hf_collision_centrality::CentralityEstimator;` to avoid typing it every time.